include/character.hpp

```diff
@@ -4,21 +4,24 @@
 #include "entity.hpp"
 #include "glob.hpp"
 
-
-class Character : public Entity {
+/**
+ * @brief Is an Entity. 
+ * 
+ */
+class Character : public Entity
+{
 public:
-	Character(const glob::vect &pos, const glob::vect &size = { 0, 0 })
-			: Entity(pos, size) {};
+	Character(const glob::vect &pos, const glob::vect &size = {0, 0})
+		: Entity(pos, size){};
+
 
 	/**
-	 * @param d the new direction of movement in radians
+	 * @param v the vector to set the direction in
 	 */
-	void set_direction(const float &d);
-
+	void set_direction(const glob::vect &v);
```

Comment:
`set_direction` taking in a vector to set the direction in makes more sense to me.